app/views/involved_persons/edit_civilian.html.erb

```diff
@@ -69,7 +69,7 @@
     <%= yes_no_question f, :resisted, "Civilian resisted?", no_label: "No (Cooperative)", class: 'required' %>
   </div>
   <div class="incident-form-row form-group additional-question" id="resistance_type_question">
-    <%= f.label :resistance_type, "Choose highest level of resistance", class: 'required' %>
+    <%= f.label :resistance_type, "Choose highest level of active resistance", class: 'required' %>
```

Comment:
I wouldn't add the word "active" in the question itself, since one of the answer options is "Passive (non-compliance)".